app/experimenter/experiments/tests/test_targeting_configs.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+from parameterized import parameterized
+from parsimonious.exceptions import ParseError
+from pyjexl.jexl import JEXLConfig
+from pyjexl.operators import default_binary_operators, default_unary_operators
+from pyjexl.parser import Parser, jexl_grammar
+
+from experimenter.experiments.constants import NimbusConstants
+
+default_config = JEXLConfig({}, default_unary_operators, default_binary_operators)
+
+
+class DefaultParser(Parser):
+    grammar = jexl_grammar(default_config)
+
+    def __init__(self):
+        super().__init__(default_config)
+
+
+class TestTargetingConfigs(TestCase):
+    def test_all_targeting_configs_defined_in_constants(self):
+        self.assertEqual(
+            set([t.value for t in NimbusConstants.TargetingConfig]),
+            set(t for t in NimbusConstants.TARGETING_CONFIGS.keys()),
+            "Targeting Configs must be defined in both NimbusConstants.TargetingConfig "
+            "and NimbusConstants.TARGETING_CONFIGS",
```

Comment:
Hmm, too bad we can't DRY these up, but I can see why there are two lists

Reply:
I know I tried, the problem is you can't define the name if you do it automatically >_<

[It is also possible to make use of the Enum Functional API with the caveat that labels are automatically generated as highlighted above](https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types)